software/postiz.yml

```diff
@@ -0,0 +1,15 @@
+name: Postiz
+website_url: https://postiz.com
+description: Self-hosted social media management software. It helps you to schedule posts, track the performance of your content, and manage all your social media accounts in one place (Alternative to Buffer, Hootsuite, Sprout Social).
+licenses:
+  - Apache-2.0
+platforms:
+  - Docker
+  - Nodejs
+tags:
+  - Analytics
+source_code_url: https://github.com/gitroomhq/postiz-app
+stargazers_count: 9249
+updated_at: '2024-10-25'
+archived: false
+
```

Comment:
```suggestion
```
These are not needed as they will be maintained automatically 👍🏻 also a extra space